webapp/src/main/java/life/qbic/usermanagement/passwordreset/PasswordResetLinkSupplier.java

```diff
@@ -11,28 +13,32 @@
  */
 class PasswordResetLinkSupplier {
```

Comment:
Nit: The class should be declared to be public for the tests to function
```suggestion
public class PasswordResetLinkSupplier {
```